src/layout/View.js

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from "react";
+import { ViewItemList } from "../view/ViewItemList";
+import { Desktop, Mobile, Tablet } from "../common/responsive";
+import { SearchText } from "../components/SearchText";
+import { SmallButton } from "../components/SmallButton";
+import { DropDown } from "../components/DropDown";
+
+export function View (){
+  const [values, setValues] = useState({
+    search : "",
+    order : "recent",
+    page : 1
+  });
+  const [popDropdown, setPopState] = useState(false);
+  
+  const handlePop = (e) => {
+    let {name, value} = e.target;
+
+    setPopState(!popDropdown);
+    handleChange(name, value);
+  }
+
+  const handleChange = (name, value) => {
+    setValues((prevValues) => ({
+      ...prevValues,
+      [name] : value,
+    }))
+  }
+  const handleInputChange = (e) => {
+    const {name, value} = e.target;
+    handleChange(name, value);
+  }
+
+  return (
+    <div className="view">
+      <section className="section-items best">
+        <div className="section-wrap">
+          <header className="section-header">
+            <h2 className="section-tit">베스트 상품</h2>
+          </header>
+          <div className="section-content">
+            <Desktop><ViewItemList order="favorite" size="4"/></Desktop>
```

Comment:
아까 봤던 screen size별 리턴값이군요 :)
조건부 렌더링이 되는게 낫지않을까요?

Reply:
제가 드린 예시를 가지고 조건부 렌더링이 된다면 이런 형식일것같아요 :)

```suggestion
            {isDesktop && (<ViewItemList order="favorite" size="4"/>)}
```